spec/features/user_can_sign_in_spec.rb

```diff
@@ -7,28 +7,31 @@
   end
 
   scenario "A user can sign in with valid credentials" do
-    visit "/"
+    visit '/'
+    click_link "Already a User?"
     fill_in :email, with: "test_email@keepingitrails.com"
     fill_in :password, with: "pa55w0rd"
     click_button "Log In"
     expect(page).to have_content("Welcome test_first_name")
   end
 
-  scenario "A user signs in with invalid email address" do
+  # not implementing the below feature in this commit:
 
-    visit "/"
+  scenario "A user signs in with invalid email address" do
+    visit '/'
+    click_link "Already a User?"
     fill_in :email, with: "test_email_other@gmail.com"
```

Comment:
can we extract all these scenarios to web helper? then we can have clear methods which reflect all the scenarios we want to test - like "correct_email_incorrect_password"